container.js

```diff
@@ -7,48 +7,49 @@ var verifier = require('alexa-verifier');
 // text string, and set a flag on the request object so other body parser
 // middlewares don't try to parse the body again
 module.exports = function alexaVerifierMiddleware(options) {
-	return function(req, res, next) {
-		if (!req.headers.signaturecertchainurl) {
-			// by default, strict header checking will not be enforced
-			if (typeof options !== 'undefined' && typeof options.strictHeaderCheck !== 'undefined' && options.strictHeaderCheck === true) {
-				// respond with a 401 error
-				res.status(401).json({ status: 'failure', reason: 'The signaturecertchainurl HTTP request header is invalid!' });
-			} else {
-				// ignore the check
-				return next();
-			}
-		}
+    return function(req, res, next) {
+        if (!req.headers.signaturecertchainurl) {
+            // by default, strict header checking will not be enforced
+            if (typeof options !== 'undefined' && typeof options.strictHeaderCheck !== 'undefined' && options.strictHeaderCheck === true) {
+                // respond with a 401 error
+                res.status(401).json({ status: 'failure', reason: 'The signaturecertchainurl HTTP request header is invalid!' });
```

Comment:
A simple what?

In my view there aren't many programming elements that are more simple and clear than an if/else statement. I really don't mind how this particular issue is fixed though. I've highlighted the problem and provided an option for the fix. The maintainer can do with it what he/she will.